tests/test_covarites_assisted_embed.py

```diff
@@ -0,0 +1,1543 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Nov 17 20:13:45 2019
+
+@author: jerryyao
+"""
+
+import unittest
+import graspy as gs
+import numpy as np
+from graspy.embed.casc import CovariateAssistedSpectralEmbed
+from graspy.embed.lse import LaplacianSpectralEmbed
+from graspy.embed.svd import selectSVD
+from graspy.simulations.simulations import er_np, er_nm, sbm
+from sklearn.cluster import KMeans
+from sklearn.metrics import adjusted_rand_score as ARI
+
+
+class TestStringMethods(unittest.TestCase):
+    def _testcasc_cca(self, *args, **kwargs):
+        A = [

```

Comment:
I would use either a graspy simulation function or somehow read in the file. Writing the adjacency matrix makes these test impossible to read.

Reply:
Yes I can definitely use a simulation with fixed seed here for the Adjacency Matrix here. But Will it be ok to leave the Covariate as they are noe